src/opnsense/www/js/opnsense_widget_manager.js

```diff
@@ -582,21 +588,28 @@ class WidgetManager  {
         ` : '';
         let $panel = $(`<div class="widget widget-${identifier}"></div>`);
         let $content = $(`<div class="widget-content"></div>`);
+        const widget = this.widgetClasses[identifier];
+        const headerStyle = widget && !widget.isTitleVisible() ? ' style="margin: 0;"' : '';
+        const titleElement = widget && widget.isTitleVisible() ? 
+        `<div id="${identifier}-title" class="widget-title"><b>${title}</b></div>` : '';
+        const linkHandle = widget && widget.isTitleVisible() ? link : '';
         let $header = $(`
-            <div class="widget-header">
+            <div class="widget-header"${headerStyle}>
                 <div class="widget-header-left"></div>
-                <div id="${identifier}-title" class="widget-title"><b>${title}</b></div>
+                ${titleElement}
                 <div class="widget-command-container">
-                    ${link}
+                    ${linkHandle}
```

Comment:
Aren't the titleElement and linkHandle non-existent for the picture here?

If so, $('.widget-title').show(); won't show anything in this case. Probably best to control the visiblity through display: none or hide() so the elements exist in all cases.